food-waste-express-api/routes/home.js

```diff
@@ -20,7 +20,7 @@ router.get("/addtobasket", async (req, res) => {
     const currentUserId = req.headers["current_user_id"]
     try {
         let userBasket = await getUserBasket(currentUserId) //gets the Basket object with the currentuserId
-        let userBasketOfIds = await userBasket.get("basketOfProductId") //returns the array of ids
+        let userBasketOfIds = await userBasket.get("basketOfProducts") //returns the array of ids
```

Comment:
Can we change this to:

let productIDsInUserBasket =  userBasket.getAllProductIDs()

1. it is clearer for the reader what is going on
2. Safer to make a function for this rather than case on a string input